src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -91,7 +99,28 @@ export class GridOutputComponent extends AngularDisposable implements IMimeCompo
 		this._cellOutput = value;
 	}
 
+	@Input() set batchId(value: number) {
+		this._batchId = value;
+	}
+
+	@Input() set id(value: number) {
+		this._id = value;
+	}
+
+	@Input() set queryRunnerUri(value: string) {
+		this._queryRunnerUri = value;
+	}
+
 	ngOnInit() {
+		let config = this.configurationService.getValue(NotebookConfigSectionName);
+		if (config) {
+			let maxRows = config[MaxTableRowsConfigName] ? config[MaxTableRowsConfigName] : undefined;
+			if (maxRows && maxRows > 0) {
+				this._configuredMaxRows = maxRows;
+			}
```

Comment:
This was likely moved from somewhere else, but we should default to a value for the `else`.

Reply:
What would be a good default value for max rows?

Reply:
5000